updater/lib/dependabot/sentry.rb

```diff
@@ -1,29 +1,31 @@
 # frozen_string_literal: true
 
-require "raven"
-
 # ExceptionSanitizer filters potential secrets/PII from exception payloads
-class ExceptionSanitizer < Raven::Processor
+class ExceptionSanitizer
   REPO = %r{[\w.\-]+/([\w.\-]+)}
   PATTERNS = {
     auth_token: /(?:authorization|bearer):? (\w+)/i,
     repo: %r{api\.github\.com/repos/#{REPO}|github\.com/#{REPO}}
   }.freeze
 
-  def process(data)
-    return data unless data[:exception] && data[:exception][:values]
+  def process_sentry(event, hint)
+    return event unless hint[:exception]
+
+    # TODO: Sentry can report multiple exceptions in an event. Our use case only expects one, esp since we report
+    # synchronously but we should probably guard against multiple...
 
-    data[:exception][:values] = data[:exception][:values].map do |e|
+    event.exception.values = event.exception.values.map do |e|
       PATTERNS.each do |key, regex|
-        next unless (matches = e[:value].scan(regex))
+        next unless (matches = e.value.scan(regex))
 
         matches.flatten.compact.each do |match|
-          e[:value] = e[:value].gsub(match, "[FILTERED_#{key.to_s.upcase}]")
+          # TODO: Discuss with upstream, because e.value is intended to be read-only, so have to hack around it with `instance_variable_set()`
+          e.instance_variable_set(:@value, e.value.gsub(match, "[FILTERED_#{key.to_s.upcase}]"))
```

Comment:
Opened upstream feature / pull requests:
* https://github.com/getsentry/sentry-ruby/issues/2071
* https://github.com/getsentry/sentry-ruby/pull/2072

Reply:
It's merged, so I updated the PR, but can't resolve this TODO until it lands it an actual release and that release gets pulled into this PR...